src/redux/actions/bulkConcepts/index.js

```diff
@@ -26,7 +25,9 @@ const fetchSourceConcepts = url => async (dispatch) => {
     dispatch(isSuccess(response.data, FETCH_SOURCE_CONCEPTS));
     dispatch(isSuccess(false, IS_LOADING));
   } catch (error) {
-    dispatch(isErrored(error.response.data, FETCH_SOURCE_CONCEPTS));
+    if (error && error.response) {
+      notify.show('Failed to fetch source concepts', 'error', 3000);
```

Comment:
Any reason we don't make use of the actual error message? I think the convention was to check if there is an error message we can show and show it, or show this generic error otherwise.

Reply:
I have modified it to check for error messages before it displays a custom one.